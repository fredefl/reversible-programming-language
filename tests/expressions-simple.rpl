test += 10;
hast += 5;
test += 2 + hast;
test -= 4*2;
print test;
print hast;