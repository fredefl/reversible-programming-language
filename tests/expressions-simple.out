9
5
3
1
-2